src/qtism/data/storage/xml/marshalling/RubricBlockMarshaller.php

```diff
@@ -131,10 +138,14 @@ protected function unmarshall(DOMElement $element)
 
                 if ($cpt instanceof Stylesheet) {
                     $stylesheets[] = $cpt;
-                } elseif ($cpt instanceof FlowStatic && !in_array($cpt->getQtiClassName(), ['hottext', 'feedbackBlock', 'feedbackInline', 'rubricBlock', 'infoControl'])) {
+                } elseif ($cpt instanceof FlowStatic && !in_array(
+                        $cpt->getQtiClassName(),
+                        ['hottext', 'feedbackBlock', 'feedbackInline', 'rubricBlock', 'infoControl']
+                    )) {
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place?